pkg/registry/core/node/strategy_test.go

```diff
@@ -126,61 +122,10 @@ func TestDropFields(t *testing.T) {
 			node:        makeNode([]string{"2000::/10", "10.0.0.0/8"}, false, false),
 			compareNode: makeNode([]string{"2000::/10", "10.0.0.0/8"}, false, false),
 		},
-		{
```

Comment:
If we should be clearing in all cases, include a test case that validates we do the right thing.